AuthSamples/SwiftSample/ViewController.swift

```diff
@@ -356,7 +387,11 @@ final class ViewController: UIViewController, UITextFieldDelegate {
         action.requiresPassword
     passwordInputLabel.alpha = isPasswordEnabled ? 1.0 : 0.6
     passwordField.isEnabled = isPasswordEnabled
-    phoneField.isEnabled = credentialType.requiresPhone
+    var userActionRequiresPhone : Bool = false
+    if action is UserAction {
```

Comment:
If you add `requiresPhone` method on the `Action` protocol you don't need to do type check here, as well as avoiding using `var`.